sdks/python/apache_beam/__init__.py

```diff
@@ -63,20 +63,16 @@
 """
 
 
-import sys
-
-
-if sys.version_info.major != 2:
```

Comment:
I believe it is important to make this check before the apache_beam import. This would avoid an import related error due to and old Python version happening before we got a chance to raise this version error.